CMakeLists.txt

```
@@ -0,0 +1,212 @@
+cmake_minimum_required(VERSION 2.8 FATAL_ERROR)
```

Otherwise, this will emit warnings on current CMake versions.
```suggestion
cmake_minimum_required(VERSION 2.8...3.5 FATAL_ERROR)
```